packages/@okta/vuepress-site/docs/guides/sign-into-mobile-app-redirect/main/ios/configmid.md

```diff
@@ -1,3 +1,5 @@
+The iOS SDK uses information from the application integration you created earlier to connect to the Okta org. The SDK loads these values from a configuration file named `Okta.plist` if it exists in your main bundle. You may also specify the values in code using `WebAuthentication(issuer:clientId:scopes:responseType:redirectUri:logoutRedirectUri:additionalParameters:)`.
+
 Create the Okta configuration file and add the keys and values for your application integration:
```

Comment:
Not really. by default it will be created in the active directory, which is the top level of the project contents. However, all plist files are moved into the default bundle during compilation unless they are specifically assigned to a different bundle.

Reply:
ok - then never mind.